model/command_autocomplete.go

```diff
@@ -75,16 +79,19 @@ type AutocompleteDynamicListArg struct {
 
 // AutocompleteSuggestion describes a single suggestion item sent to the frontend
 type AutocompleteSuggestion struct {
-	// Hint describes what user might want to input
+	// Suggestion describes what user might want to input
```

Comment:
An example in the comment here would go a long way... not 100% intention-revealing Suggestion/Hint/Description - what is what